engine/engine/content/builtins/render/default.render_script

```diff
@@ -212,6 +250,7 @@ end
 function on_message(self, message_id, message)
     local state = self.state
     local camera = state.main_camera
+
     if message_id == MSG_CLEAR_COLOR then
         update_clear_color(state, message.color)
     elseif message_id == MSG_WINDOW_RESIZED then
```

Comment:
so we can remove `MSG_SET_VIEW_PROJ` and `MSG_SET_CAMERA_PROJ` in `on_message` and in our docs as well.

Reply:
This will break camera libraries that do not have a camera component but send these messages pretending to be a camera.

Reply:
Ah, ok